fdbserver/storageserver.actor.cpp

```diff
@@ -8819,10 +8819,8 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 	                                             data->counters.bytesFetched,
 	                                             data->counters.kvFetched);
 
-	// Set read options to use non-caching reads and set Fetch type unless low priority data fetching is disabled by
-	// a knob
-	state ReadOptions readOptions = ReadOptions(
-	    {}, SERVER_KNOBS->FETCH_KEYS_LOWER_PRIORITY ? ReadType::FETCH : ReadType::NORMAL, CacheResult::False);
```

Comment:
Do we want to always set the option as ReadType::FETCH?

Reply:
Yes, is won't affect other storage engines.